qiskit/pulse/pulse_lib/parametric_pulses.py

```diff
@@ -26,7 +26,8 @@
   - be a well known and/or well described formula (include the formula in the class docstring)
   - take some parameters (at least `duration`) and validate them, if necessary
   - implement a `get_sample_pulse` method which returns a corresponding SamplePulse in the
-    case that it is assembled for a backend which does not support it.
+    case that it is assembled for a backend which does not support it. Ends are zeroed to more
+    closely reflect the behavior of the backend.
```

Comment:
I think qiskit pulse is backend agnostic. Can we just say like zeroed to avoid steep jump in pulse edges?

Reply:
Fixed in https://github.com/Qiskit/qiskit-terra/pull/4640/commits/e2082984e0793392b7a0d257fd8ed6b6d0a4a20f.